androidApp/src/androidTest/java/com/projectnewm/interactions/NewmAppInteractions.kt

```diff
@@ -0,0 +1,63 @@
+package com.projectnewm.interactions
+
+import androidx.compose.ui.test.SemanticsNodeInteraction
+import androidx.compose.ui.test.assertIsDisplayed
+import androidx.compose.ui.test.junit4.ComposeTestRule
+import androidx.compose.ui.test.onNodeWithTag
+import com.projectnewm.screens.home.TAG_HOME_SCREEN
+import com.projectnewm.screens.stars.TAG_STARS_SCREEN
+import com.projectnewm.screens.tribe.TAG_TRIBE_SCREEN
+import com.projectnewm.screens.wallet.TAG_WALLET_SCREEN
+
+fun onNewmApp(
+    composeTestRule: ComposeTestRule,
+    actions: NewmAppInteractions.() -> Unit
+) {
+    actions(NewmAppInteractions(composeTestRule))
+}
+
+class NewmAppInteractions(private val composeTestRule: ComposeTestRule) {
+    private val walletScreenInteraction: SemanticsNodeInteraction
+        get() = composeTestRule.onNodeWithTag(TAG_WALLET_SCREEN)
+
+    private val starsScreenInteraction: SemanticsNodeInteraction
+        get() = composeTestRule.onNodeWithTag(TAG_STARS_SCREEN)
+
+    val tribeScreenInteraction: SemanticsNodeInteraction
+        get() = composeTestRule.onNodeWithTag(TAG_TRIBE_SCREEN)
+
+    val homeScreenInteraction: SemanticsNodeInteraction
+        get() = composeTestRule.onNodeWithTag(TAG_HOME_SCREEN)
```

Comment:
used tags because the content of the screens will change and their content is outside the scope of these tests

Reply:
That will probably be the best way to go for now, just in case design changes 👍🏼